README.md

```diff
@@ -1,103 +1,30 @@
-
-![2 standalones diagram](https://www.lucidchart.com/publicSegments/view/4f01dc86-c34a-49a9-b619-f3d1056e7a41/image.png)
+![Architecture Diagram](https://www.lucidchart.com/publicSegments/view/4f01dc86-c34a-49a9-b619-f3d1056e7a41/image.png)
 
 ## How it works
 
-* All of the Chef Server and Chef Automate services have been packaged as Habitat `.hart` files here: [https://bldr.habitat.sh/#/pkgs/chef-server]
-* and exported to docker containers here:  [https://hub.docker.com/r/chefserverofficial/]
-* These containers support running under a random, arbitrary user and/or group id.
-* Launch containers via `docker-compose` or `docker run ...`  Compose files have been created which pull the docker containers and run them using Host mode networking. Alternatively, [docker run Chef Server](https://github.com/chef-customers/dockerized-chef-services/tree/master/docker_run_chef_server.md) and [docker run Automate](https://github.com/chef-customers/dockerized-chef-services/tree/master/docker_run_automate.md) files demonstrate how to launch via `docker run ...` commands.
+* All of the Chef Server and Chef Automate services have been packaged as Habitat `.hart` files
+* The hab packages have been exported to docker containers and published to docker hub
+* These containers support running under a random, arbitrary effective user and/or group id
+* Launch containers via the included [Chef Server](https://github.com/chef-customers/dockerized-chef-services/tree/master/terraform/chef-server.sh) or [Automate Server](https://github.com/chef-customers/dockerized-chef-services/tree/master/terraform/automate-server.sh) scripts, or alternatively via `docker run ..`
 * Each host forms its own non-permanent Habitat gossip ring, sharing service discovery data intra-host only
 * Environment variables are used to configure settings from the default
 
 ## Deploying it
 
 ### Using AWS and Terraform
 
-Customize the `main.tf` [Terraform](terraform.io) configuration file and deploy with `terraform apply`
-
-### Manually
-
-1. Provision 2 hosts that are running a modern Linux OS (RHEL 7 or Ubuntu 16.04) as well as recent versions of Docker and optionally docker-compose
-2. Attach SAN (block) storage to each host for storing persistent data
-3. Read [Deployment Notes](https://github.com/chef-customers/dockerized-chef-services#deployment-notes)
-
-### Data Directories
-
-Each Host must have directories created under `$DATA_MOUNT` and be owned by `$USER_ID`:
-
-For Automate 1.x:
-
-* postgresql
-* rabbitmq
-* elasticsearch
-* maintenance
-* workflow
-* compliance
-* nginx
-
-For Chef Server:
+Customize the `terraform.tfvars.example` [Terraform](terraform.io) configuration file, rename it to `terraform.tfvars` and deploy with `terraform apply`
```

Comment:
Weird, but I just noticed that `[Terraform](terraform.io)` doesn't quite do what we thought it does.   needs an `https://` added.